### PR TITLE
Home page & contact page

### PR DESCRIPTION
**Pull request by umairwaheed**

### CODE REVIEW

File: v2/prixite/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            ))}
+          </Grid>
+        </Box>
+      </Container>
+      <Footer />
+    </>
+  );
+}
+
+const services = [
```
Comment (umairwaheed) [inline v2/prixite/pages/index.js:189]: Please move the global variables to the top after the imports.

File: v2/prixite/pages/_app.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import "../styles/globals.css";
```
Comment (umairwaheed) [inline v2/prixite/pages/_app.js:1]: @aftab76ahmad76 why the name of this file begins with underscore "_"?

File: v2/prixite/pages/careers.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import Head from "next/head";
+import Header from "../components/header";
+import careersPageBg from "../public/images/careersPageBg.jpg";
+
+export default function ContactUs() {
```
Comment (umairwaheed) [inline v2/prixite/pages/careers.js:7]: Name of every component should match the name of the file. So you should create another file called "ContactUs.js" in v2/prixite/pages and import "ContactUs" from that file.

Comment (umairwaheed) [reply v2/prixite/pages/careers.js:7]: Plus I think the name of the component should be `Careers`?

File: v2/prixite/pages/contact-us.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+import Image from "next/image";
+import Footer from "../components/footer";
+import Header from "../components/header";
+import contactUsPageBg from "../public/images/contactUsPageBg.png";
+import FacebookRoundedIcon from "@mui/icons-material/FacebookRounded";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import GitHubIcon from "@mui/icons-material/GitHub";
+import Link from "next/link";
+
+export default function ContactUs() {
```
Comment (umairwaheed) [inline v2/prixite/pages/contact-us.js:13]: The name of the component should match the file name.

File: v2/prixite/pages/index.js [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+import webDevelopment from "../public/images/webDevelopment.jpg";
+import mobileDev from "../public/images/mobileDev.jpg";
+import scraping from "../public/images/scraping.jpg";
+import pwa from "../public/images/pwa.jpg";
+import ArrowRightAltIcon from "@mui/icons-material/ArrowRightAlt";
+import reactjs from "../public/images/reactjs.jpg";
+import wordpress from "../public/images/wordpress.jpg";
+import python from "../public/images/python.jpg";
+
+export default function Home() {
```
Comment (umairwaheed) [inline v2/prixite/pages/index.js:25]: This should be Index?

File: v2/prixite/pages/_app.js (hunk 2 above)
Comment (aftab76ahmad76) [reply v2/prixite/pages/_app.js:1]: It's by default in next js

File: v2/prixite/pages/careers.js (hunk 3 above)
Comment (aftab76ahmad76) [reply v2/prixite/pages/careers.js:7]: Got it

File: v2/prixite/pages/contact-us.js (hunk 4 above)
Comment (aftab76ahmad76) [reply v2/prixite/pages/contact-us.js:13]: Got it

File: v2/prixite/pages/index.js (hunk 5 above)
Comment (aftab76ahmad76) [reply v2/prixite/pages/index.js:25]: Name: "index" for a component would be confusing,, I will make it better in other way

**Pull request merged**